doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -350,6 +350,27 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-6-1 v2-6-0
+
+KRKNWK-18965: Malformed group messages can cause memory leak
+  Matter accessories utilizing group communication may experience memory leaks if the secure group message is malformed.
+
+  **Workaround:** Manually cherry-pick commit to ``sdk-connectedhomeip`` from the upstream `Matter SDK PR #32970 <https://github.com/project-chip/connectedhomeip/commit/bdf3e6e183cba3d13bc5103bf014b47841a14de1>`
```

Comment:
I see we don't really have a standard way of wording this in the NCS, but the most common seems to be:
```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``bdf3e6e183cba3d13bc5103bf014b47841a14de1``).
```

Reply:
Here, i have added a bit more text, as the commit exists only on the upstream Matter SDK and will not be pulled into sdk-connectedhomeip. Would that work?